creator-node/src/components/contentBlacklist/contentBlacklistComponentService.ts

```diff
@@ -3,12 +3,29 @@ const models = require('../../models')
 
 const types = models.ContentBlacklist.Types
 
-const getAllTrackIds = async () => {
+interface IBlacklistedContent {
```

Comment:
To be consistent with how we do things elsewhere in our codebase, can you change your `interface`s to `type`s?

Also, no need for the prefix in front of types. `BlacklistedContent` works

Reply:
done and done